babybuddy/middleware.py

```diff
@@ -6,6 +6,12 @@
 from django.utils import timezone, translation
 from django.conf.locale.en import formats as formats_en_us
 from django.conf.locale.en_GB import formats as formats_en_gb
+from os import getenv
+from django.contrib.auth.middleware import RemoteUserMiddleware
+
+
+class CustomRemoteUser(RemoteUserMiddleware):
```

Comment:
Could you move this class down to the bottom of the file? Just a personal preference thing to have classes after functions.

Reply:
Will do.